.github/workflows/repo-health-job.yml

```diff
@@ -15,6 +15,15 @@ on:
       READTHEDOCS_API_KEY:
         description: "API Key for READ THE DOCS Access"
         required: true
+      EMAIL_ADDRESS_FOR_ALERTS:
+        description: "Email Address for the alerts"
+        required: false
+      EDX_SMTP_USERNAME:
```

Comment:
Also, we can drop the `edx` suffix from the names of these secrets.

Reply:
Done.